ldm/modules/encoders/modules.py

```diff
@@ -463,6 +463,10 @@ def forward(self, text, **kwargs):
     def encode(self, text, **kwargs):
         return self(text, **kwargs)
 
+    def set_textual_inversion_manager(self, manager): #TextualInversionManager):
+        # TODO all of the weighting and expanding stuff needs be moved out of this class
```

Comment:
i think this comment may be redundant? if i wrote it it's referring to stuff that Compel now handles

Reply:
Removed the comment.